src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8596,14 +8596,14 @@ internal enum StreamInfoLevels { FindStreamInfoStandard = 0 }
                 IntPtr lpSecurityAttributes, FileMode dwCreationDisposition,
                 int dwFlagsAndAttributes, IntPtr hTemplateFile);
 
-            [DllImport(PinvokeDllNames.FindFirstStreamDllName, ExactSpelling = true, CharSet = CharSet.Unicode, SetLastError = true)]
+            [DllImport("kernel32.dll", ExactSpelling = true, CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
Wouldn't it be better to change `FindFirstStreamDllName` to point to `kernel32.dll`?

Reply:
I believe we could preserve these constants (as reminder) if we want revert the change after Windows 8 (and 7) EOL.